blocks/story-carousel-block/features/story-carousel/default.jsx

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import PropTypes from "@arc-fusion/prop-types";
+import { Carousel, Image, Icon, Heading, Paragraph, Stack } from "@wpmedia/arc-themes-components";
+import { useContent } from "fusion:content";
+import { extractImageFromStory } from "@wpmedia/resizer-image-block";
+import { useFusionContext, useComponentContext } from "fusion:context";
+import getProperties from "fusion:properties";
+import getTranslatedPhrases from "fusion:intl";
+
+const BLOCK_CLASS_NAME = "b-story-carousel";
+
+const StoryCarousel = ({
+	customFields: {
+		inheritGlobalContent = false,
+		carouselContentConfig: { contentService = "", contentConfigValues = {} } = {},
+		headerText,
+		itemHeaderTruncationLines = 0,
+		itemDescriptionTruncationLines = 0,
+	},
+	globalContent = {},
+}) => {
+	const { id } = useComponentContext();
+	const { arcSite } = useFusionContext();
+	const { locale = "en" } = getProperties(arcSite);
+	const phrases = getTranslatedPhrases(locale);
+
+	const content = useContent({
+		source: contentService,
+		query: contentConfigValues,
+	});
+
+	const showGlobalContent =
+		typeof inheritGlobalContent === "undefined"
+			? typeof carouselContentConfig === "undefined"
+			: inheritGlobalContent;
+	const { content_elements: elements = [] } = showGlobalContent ? globalContent : content;
+
+	if (!elements.length) return null;
+
+	return (
+		<Stack className={BLOCK_CLASS_NAME}>
+			{headerText && headerText.trim() !== "" && (
+				<Heading className={`${BLOCK_CLASS_NAME}__main-title`}>{headerText}</Heading>
+			)}
+
+			{elements && elements.length && (
+				<Carousel
+					id={id}
+					label={phrases.t("story-carousel.aria-label")}
+					nextButton={
+						<Carousel.Button id={id} label={phrases.t("story-carousel.right-arrow-label")}>
+							<Icon name="ChevronRight" />
+						</Carousel.Button>
+					}
+					previousButton={
+						<Carousel.Button id={id} label={phrases.t("story-carousel.left-arrow-label")}>
+							<Icon name="ChevronLeft" />
+						</Carousel.Button>
+					}
+				>
+					{elements.map((item, index) => (
+						<Carousel.Item
+							key={item._id}
+							label={`${phrases.t("story-carousel.slide-indicator", {
+								current: index + 1,
+								maximum: elements.length,
+							})}`}
+						>
+							<a className={`${BLOCK_CLASS_NAME}__story-card`} href={item.canonical_url}>
+								<Image alt={item.headlines.basic} src={extractImageFromStory(item)} />
```

Comment:
alt should be a blank string is if there is a heading or description otherwise there will be duplicated text for assistive technologies

Reply:
Done